packages/backend/src/formatter.ts

```diff
@@ -0,0 +1,56 @@
+import { Explore, fieldId as getFieldId, getFields, ResultRow } from 'common';
+
+function formatValue(format: string, value: any): any {
```

Comment:
In the app there are a few places where we format undefined values as ∅ - it could be nice to add that logic here too!

Reply:
This would also prevent things like `undefined km` or `$undefined`